changes/serializers.py

```diff
@@ -127,3 +129,17 @@ class Data(serializers.Serializer):
         address = PhoneNumberField(country_code="ZA")
 
     data = Data()
+
+
+class SeedMessageSenderFailedMsisdnHookSerializer(serializers.Serializer):
+    class Hook(serializers.Serializer):
+        id = serializers.IntegerField()
+        event = serializers.ChoiceField(choices=["identity.no_address"])
+        target = serializers.CharField()
+
+    hook = Hook()
+
+    class Data(serializers.Serializer):
+        identity = str(uuid.uuid4())
```

Comment:
I don't think that this will validate that `identity` is a UUID, this needs to be an instance of the UUIDField: https://www.django-rest-framework.org/api-guide/fields/#uuidfield